src/ActionModal/components/SendModal.vue

```diff
@@ -360,6 +366,7 @@ export default {
       amount: {
         required: x => !!x && x !== `0`,
         decimal,
+        max: x => Number(x) <= this.maxAmount,
```

Comment:
Why add this if we already have between?

Reply:
To show a better error message if the max has passed. Maybe instead we change the between error to show the better error

Reply:
This is why I was asking. They overlap

Reply:
yep! we should probably remove `between` in favor of individual ones to show better errors?

Reply:
but actually just with `max` we would get all the possible errors covered right?